tools/bridge-deploy/bridge_deploy/core.py

```diff
@@ -0,0 +1,56 @@
+from typing import Dict
+from web3 import Web3
+from web3.contract import Contract
+from deploy_tools.deploy import deploy_compiled_contract
+
+from bridge_deploy.utils import load_build_contract
```

Comment:
Suggestion: As this is the only load method we're going to use in this tool, maybe rename to `load_contract`, or go the full `load_contracts_json` route as seen in the auction deploy tool.

Reply:
Good point. Also obsolete part. I would move the whole function into the core module. There is nothing to share between multiple modules anymore. :+1:

Reply:
:heavy_check_mark: